lib/package.json

```diff
@@ -12,15 +12,20 @@
     "dist",
     "index.d.ts"
   ],
+  "source": "index.js",
   "main": "dist/index.js",
+  "module": "dist/index.mjs",
+  "unpkg": "dist/index.umd.js",
   "typings": "./index.d.ts",
   "scripts": {
-    "build": "webpack --config webpack.config.js"
+    "build": "microbundle --jsx React.createElement"
```

Comment:
```sh
ERROR in ../lib/dist/index.mjs 1:3393-3394
Can't import the named export 'Component' from non EcmaScript module (only default export is available)
 @ ./src/modules/example.js
 @ ./src/containers/index.js
 @ ./src/index.js
```
https://github.com/developit/microbundle/issues/426

Reply:
fixed by renaming **module** dist from `dist/index.mjs` to `dist/index.esm.js`.
Reason webpack treat's `*.mjs` modules very strictly in regard to named imports of commonJS modules.